metrics.py

```diff
@@ -0,0 +1,40 @@
+from prometheus_client import Gauge
+import time
+
+# Assuming the structure of the data based on the provided example
+# We will create a Gauge for each field in the blob data
+
+def create_metrics():
+    """
+    Create Prometheus Gauges for various fields in the blob data.
+    """
+    metrics = {
+        'blob_index': Gauge('blob_index', 'Index of the blob'),
+        'reference_block_number': Gauge('reference_block_number', 'Reference block number'),
+        'batch_id': Gauge('batch_id', 'Batch ID'),
+        'confirmation_block_number': Gauge('confirmation_block_number', 'Confirmation block number'),
+        'time_difference': Gauge('time_difference_hours', 'Difference in hours from the requested_at time to current time')
+        # Additional metrics can be added here based on the data fields
+    }
+    return metrics
+
+def update_metrics(metrics, data):
+    """
+    Update the Prometheus metrics with the latest data from the API.
+
+    Args:
+    metrics (dict): The dictionary of Prometheus Gauges.
+    data (dict): The data fetched from the API.
+    """
+    current_time = time.time()
+    for blob in data.get('result', {}).get('data', {}).get('json', {}).get('data', []):
+        requested_at = blob.get('requested_at', 0)
+        time_difference = (current_time - requested_at) / 3600  # Convert seconds to hours
```

Comment:
I recommend not converting the `time_difference` to hours. This conversion can be done with PromQL in the query without reducing the precision of the data source.

Reply:
Good recommendation. Done [87ef4e8](https://github.com/NethermindEth/eigenda-blob-scrapper/pull/1/commits/87ef4e86bc30b520cd6b6d93985a98fe477f2415)